.idea/misc.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="MavenProjectsManager">
```

Comment:
this loads Maven into this project, is this intentional?

Reply:
From our last conversation Mariya, is the version that had the app package?